src/Rules/Deprecations/TypeHintDeprecatedInClassMethodSignatureRule.php

```diff
@@ -51,7 +52,7 @@ public function processNode(Node $node, Scope $scope): array
 						strtolower($deprecatedClass->getClassTypeDescription()),
 						$deprecatedClass->getName(),
 						$this->deprecatedClassHelper->getClassDeprecationDescription($deprecatedClass),
-					))->identifier(sprintf('parameter.deprecated%s', $deprecatedClass->getClassTypeDescription()))->build();
+					))->identifier(sprintf('parameter.deprecated%s', $deprecatedClass->getClassTypeDescription()))->line($line)->build();
```

Comment:
Ideally, this should report at the parameter line, but that would be much bigger BC.